src/services/config.js

```diff
@@ -0,0 +1,93 @@
+const httpStatusCode = require('@generics/http-status')
+const common = require('@constants/common')
+const organizationExtensionsQueries = require('@database/queries/organizationExtensions')
+const { UniqueConstraintError, ForeignKeyConstraintError } = require('sequelize')
+const { Op } = require('sequelize')
+const responses = require('@helpers/responses')
+const _ = require('lodash')
+const { Organizations } = require('aws-sdk')
+
+module.exports = class configsHelper {
+	/**
+	 * List Configs.
+	 * @method
+	 * @name list
+	 * @param {Integer} organization_id
+	 * @param {String} id -  id.
+	 * @returns {JSON} - List of configs as response.
+	 */
+
+	static async list(organization_id) {
+		let orgExtenstionData = {}
+		let configData = []
+		try {
+			// define filter
+			const filter = {
+				organization_id,
+			}
+			// attributes to fetch from organisation Extenstion
+			const attributes = ['review_required', 'show_reviewer_list', 'min_approval', 'resource_type', 'review_type']
+			// fetch the current list of resources
+			const resourceList = common.RESOURCE_LIST
+
+			// instance level configurations from env as default configs
+			const default_configs = {
+				review_required: process.env.REVIEW_REQUIRED == 'true' ? true : false,
+				show_reviewer_list: process.env.SHOW_REVIEWER_LIST == 'true' ? true : false,
+				min_approval: Number(process.env.MIN_APPROVAL),
+				review_type:
+					process.env.REVIEW_TYPE.toLowerCase == 'sequential'
```

Comment:
@adithyadinesh0412 define 'sequential' in the commin file, it should be in capital letters